flink-core/src/main/java/org/apache/flink/configuration/AkkaOptions.java

```diff
@@ -28,31 +28,143 @@
 @PublicEvolving
 public class AkkaOptions {
 
+	public static String DEFAULT_AKKA_TRANSPORT_HEARTBEAT_INTERVAL = "1000 s";
+
+	public static String DEFAULT_AKKA_TCP_TIMEOUT = "20 s";
+
+	public static String DEFAULT_AKKA_WATCH_HEARTBEAT_INTERVAL = "10 s";
```

Comment:
These should be moved into the `defaultValue` clause of the config option. They can be accessed from the ConfigOption using `ConfigOption#defaultValue()`.